version.py

```diff
@@ -0,0 +1,81 @@
+# Authors: Douglas Creager <dcreager@dcreager.net>
```

Comment:
Please don't commit copied code into the repository.

It's already available in `golem-messages` anyway.

Reply:
:+1: I will use the golem_message version then, but I have used it only because Darek's proposed it himself.

Reply:
It's just a script that prints the version so it does not need to be importable.

Reply:
So what are you suggesting to do ?

Reply:
Just don't include this script here. We'll call it directly in deployment.

For convenience you might just add a helper script that detects the location of golem-messages and calls the script from there.

Reply:
:+1: I will add script but please note this is returned with every response so everyone need to run it on their environments to perform a request without failing.